CreativeFitz/Summary
# Bf ticket2

# Description

When User is signed in they should land on the course library page which shows all courses all users have created.


## Related Ticket(s)
Ticket 2

## Steps to Test
-Run in browser
-Sign in to app
-After clicking sign in you should see courses library page.


# Checklist:
- [n/a] I have commented my code, particularly in hard-to-understand areas
- [n/a] I have made corresponding changes to the documentation
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [n/a] New and existing unit tests pass locally with my changes
